spec/features/user_adds_vendor_spec.rb

```diff
@@ -15,8 +17,18 @@
     member = FactoryGirl.create(:user)
     sign_in(member)
     visit vendors_path
-
     fill_in 'Vendor Name', with: 'Bacon Truck'
+    select('Breakfast', from: 'vendor_category_ids')
+    click_button 'Add Vendor'
+    expect(page).to have_content('Vendor added. Pending review.')
+  end
+
+  scenario 'admin tries to add a new vendor' do
+    member = FactoryGirl.create(:user, user_type: 'Admin')
```

Comment:
`member` seems to be referring to a non-admin user above - so call this variable `admin` to be more clear!